src/reducers/profile-view.js

```diff
@@ -276,6 +277,17 @@ const viewOptionsPerThread: Reducer<ThreadViewOptions[]> = (
         ...state.slice(threadIndex + 1),
       ];
     }
+    case 'CHANGE_RIGHT_CLICKED_MARKER': {
```

Comment:
Observation: I made a similar comment in the other patch. Maybe as a follow-up we could collect all the context menu state in a single place in the store, and not in the thread view options. Part of my thought here is that we only have one context menu open at a time, so it's impossible to store these on each thread like a selected call node.

Reply:
good point.
But the value we store is relative to the selected thread. So we'll need to keep that in mind. Having it in thread view options makes it more explicit, but I don't have a strong feeling otherwise.